### DISCUSSION TITLE
Is it possible to use it from PowerShell?

### DISCUSSION THREAD

**Discussion by pyetrosafe-hnd** [CharsetDetector/UTF-unknown#173] [Q&A] [answered]
Is it possible to use it from PowerShell?
I created a script that converts files from ISO-8859-1 to UTF-8, but I'm using Git\bin\bash.exe to identify the Charset.
I think it would be more efficient and faster to use this lib instead.

**Comment by 304NotModified** [comment, marked as answer]
Yes that should work, see https://stackoverflow.com/questions/3079346/how-to-reference-net-assemblies-using-powershell
